meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-core.bb

```diff
@@ -27,6 +27,8 @@ RDEPENDS:${PN} = "\
     kanto-auto-deployer \
     leda-utils \
     openvpn \
+    gpsd \
```

Comment:
This has been done on purpose, so that someone using the distro can have a plug and play experience.

The gpsd is not configured with any specific gps device or serial connection, so it works without having the necessary hardware. If a device is connected, it will be automatically used.

I could add a distro feature, but I think it's not necessary.

Reply:
If that's the case, it's fine for gpsd to stay in core.